specs/DesignDiscussions/InputAccessibility.md

```diff
@@ -0,0 +1,70 @@
+# Input Accessibility
+
+## Error Accessibility
+
+How should a renderer present error messages to screen readers for inputs that have failed validation?
+
+### On Focus
+
+What is announced when focus moves to a validated input?
+
+1. When an Input is in focus, announce the input's label. If the input has failed validation, **append** the error message.
```

Comment:
@golddove to what are we appending the error message corresponding to this first failed validation input ?

Reply:
Can you pull the "decision" option up to the top of the spec as our plan of action so its very clear ? Does the statement above still apply to Option 1 ?

Reply:
Sorry i just realized each option seems to be an option and a decision?